dalite/settings.py

```diff
@@ -423,6 +428,10 @@
 # External framing
 FRAMING_ALLOWED_FROM = ["*"]
 
+CSP_INCLUDE_NONCE_IN = []
```

Comment:
Were the latest dev changes merged in this branch?  These settings have been changed.

Reply:
Ok, I can see the merge in the commit history.  But these settings seem to have been added back.  Is there a reason?

Reply:
It's probably because they were added at different times in the two branches and so when merged kept in.

Reply:
That is unfortunate... is there a workflow to avoid this problem?

Reply:
(There may be a handful of changes implemented in `dev` to get CSP working that have been overwritten... that would be my nightmare!)

Reply:
I'll do a diff between the two branches on that file to make sure `async-report` is up to date.
Is there any other file that's impacted?

Reply:
Not that I noticed.  But there are a lot of files this time around.